server/api/rounds.py

```diff
@@ -334,7 +335,7 @@ def calculate_risk_measurements(election: Election, round: Round):
                 sampled_ballot_interpretations_to_cvrs(contest),
             )
 
-        round_contest.end_p_value = p_value
+        round_contest.end_p_value = float(p_value)
```

Comment:
Could the `risk_limit` passed to `compute_risk` remain a `float` and get computed into a `Decimal` internally? It doesn't seem like the abstraction needs to change

Similarly, could the resulting p-value be a `float` as well, since it just gets converted immediately after being returned from `compute_risk`?

(Also, you can extend this question to all functions that are part of the external interface presented by the audit math)

Reply:
I changed `risk_limit` because it was super cumbersome to have to cast it in every function. This way we're just casting it once. 

`p_value` needs to be computed as a Decimal to avoid losing precision. I could return it as a float, that's fair.